lib/fizzy/limits.hpp

```diff
@@ -4,13 +4,20 @@
 
 #pragma once
 
+#include <cstddef>
```

Comment:
Why is this needed?

Reply:
Leftover from the version returning size_t, will remove.